starlark/testdata/math.star

```diff
@@ -0,0 +1,43 @@
+# Tests of math module.
+
+load('math.star', 'math')
+load('assert.star', 'assert')
+
+def assert_near(actual, desired, threshold):
+  val = desired - actual
+  if val < 0:
+    val = val * -1
+  return val < threshold
+
+assert.eq(math.ceil(0.5), 1.0)
+assert.eq(math.fabs(-2.0), 2.0)
+assert.eq(math.floor(0.5), 0)
+
+assert.eq(math.exp(2.0), 7.38905609893065)
+assert.eq(math.sqrt(4.0), 2.0)
+
+assert.eq(math.acos(1.0), 0)
+assert.eq(math.asin(1.0), 1.5707963267948966)
```

Comment:
Ideally the test cases would be expressed to provide some mathematical insight,
e.g. `sin(math.pi / 6) = 0.5`